robyn/openapi.py

```diff
@@ -423,7 +423,7 @@ def get_openapi_docs_page(self) -> FileResponse:
         """
         with resources.open_text("robyn", "swagger.html") as path:
             html_file = path.read()
-        return html(html_file)
+        return FileResponse(html_file, headers=Headers({"Content-Type": "text/html"}))
```

Comment:
why not just return `html` ??

Reply:
The method signatures don't match
```
def get_openapi_docs_page(self) -> FileResponse:
...
def html(html: str) -> Response:
```

Reply:
Ah, my bad. Can we just return `html` then and use the typing as `Response`. Apologies for all the back and forth here.

Last comment btw.

Reply:
I hope I've understood this properly.

"use the typing as `Response`"

You mean change `get_openapi_docs_page` return type to Response?

Reply:
Yes @dave42w :D

Reply:
Done :-)